pkg/apk/impl/sign.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2022 Chainguard, Inc.
```

Comment:
we should upstream this to alpine-go

Reply:
Just the sign changes? Or the whole `pkg/apk/impl`?

Reply:
I would say at least the signing, but we should do it incrementally.

Reply:
We had a bit of a discussion on slack on this. I think the consensus was not to hold out for upstreaming, but rather to do it in chainguard-dev org, and then eventually upstream it.